cvmfs/swissknife_lease.cc

```diff
@@ -55,83 +54,85 @@ int CommandLease::Main(const ArgumentList& args) {
   params.repo_service_url = *(args.find('u')->second);
   params.action = *(args.find('a')->second);
   params.user_name = *(args.find('n')->second);
-  params.lease_fqdn = *(args.find('p')->second);
+
+  const std::string lease_full_path = *(args.find('p')->second);
+  size_t delimiter_position = lease_full_path.find_first_of('/');
```

Comment:
I think the `SplitString()` from util/string or `GetFileName()` / `GetParentPath()` from util/posix could be used here.